resources/server/messages/messages.db.ts

```diff
@@ -107,11 +106,15 @@ export class _MessagesDB {
    */
   async createLabel(userIdentifier: string, groupId: string, label: string): Promise<void> {
```

Comment:
Lets rename this method to updateLabel or something of that sort as we no longer are creating anything